docs/.vuepress/config.js

```diff
@@ -272,6 +272,7 @@ module.exports = {
           children: [
             'plugins/chart-area-border',
             'plugins/quadrants',
+            'plugins/doughnut-empty-state',
```

Comment:
I think it makes more sense to the consumer to have these sorted alphabetically rather than in order of addition.

Reply:
I was thinking just the plugin samples, but ok!
I'll have a look if everything is ok this way.

Reply:
Yeah I know, but I wanted to do it for some time since it makes searching easyer and thought if I am doing plugins lets do it good and change everything to be easyer to look through :)